fivetran_provider_async/triggers/fivetran.py

```diff
@@ -28,7 +28,7 @@ def __init__(
         polling_period_seconds: float,
```

Comment:
Do we need both `polling_period_seconds` and `poll_interval` in the trigger? Its not regarding to your changes but can we fix this ?

Reply:
Fixed to use `poke_interval` everywhere [a430049](https://github.com/astronomer/airflow-provider-fivetran-async/pull/8/commits/a4300496e12aeba5d6673638067638d8ecdee9cc)